pom.xml

```diff
@@ -189,6 +189,12 @@
             <version>4.2.0.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Was this needed just to support the new test? I'm surprised to see it given there doesn't appear to be any xml-specific code added.

Reply:
I was running with Java 11 which threw class not found for `javax.xml.bind.ValidationException` so I added the dependency. It works either way w/ Java 8 so I would say we leave it for future compatibility.